### ISSUE TITLE
indent rule not working with in highly-wrapped context

### ISSUE THREAD

**Issue by Trott** [eslint/eslint#7735]
**Tell us about your environment**

* **ESLint Version:** current master
* **Node Version:** 7.2.1
* **npm Version:** 3.10.10

**What parser (default, Babel-ESLint, etc.) are you using?**

default

**Please show your full configuration:**
n/a; no configuration file

```




```

**What did you do? Please include the actual source code causing the issue.**

command line: `eslint --no-eslintrc test.js --rule 'indent: [2, 2, {ArrayExpression: first, ObjectExpression: first}]'`

```js
var commands = [
  { 'method': 'Debugger.setBreakpointByUrl',
    'params': { 'lineNumber': 5,
                'url': session.mainScriptPath,
                'columnNumber': 0,
                'condition': ''
              }
  },
  { 'method': 'Debugger.resume'},
  [ { 'method': 'Debugger.getScriptSource',
      'params': { 'scriptId': session.mainScriptId } },
    expectMainScriptSource ],
];
```

**What did you expect to happen?**

no error reported

**What actually happened? Please include the actual, raw output from ESLint.**
```console
/Users/trott/test/test.js
  7:15  error  Expected indentation of 4 spaces but found 14  indent

✖ 1 problem (1 error, 0 warnings)
```

**Comment by Trott** [comment]
Actually, looking more closely, this looks correct. Never mind!
